docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+In the event that the Primary Domain Validation Determination or Primary CAA Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table, CAs MAY immediately retry Multi-Perspective Domain Validation using the same validation method or an alternative method (e.g., a CA can immediately retry validation using “Email to DNS TXT Contact” if “Agreed-Upon Change to Website - ACME” does not corroborate the outcome of Multi-Perspective Domain Validation). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+Phased Implementation Timeline:
+- *Effective March 15, 2024*, the CA SHOULD implement Multi-Perspective Domain Validation as described in this section. 
+- *Effective September 15, 2024*, the CA MUST implement Multi-Perspective Domain Validation as described in this section; however, the CA MAY proceed with certificate issuance if the Primary Domain Validation Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table.
+- *Effective March 15, 2025*, the CA MUST implement Multi-Perspective Domain Validation as described in this section; however, the CA MUST NOT proceed with certificate issuance if the Primary Domain Validation Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table and the "Defining Corroborating CAA Evidence" table.
+
+Even though the "# of Network Perspectives and Quorum Requirements" table permits issuance when some Network Perspectives do not corroborate the Primary Domain Validation Determination, the set of Network Perspectives that do corroborate the Primary Domain Validation Determination MUST be spread over two distinct continents for a CA to proceed with certificate issuance after March 15, 2025.
+
+Table: # of Network Perspectives and Quorum Requirements
+
+| Minimum # of Network Perspectives that MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination | Maximum # of Network Perspectives that MAY NOT corroborate the Primary Domain Validation Determination and Primary CAA Determination | Description |
+|---|---|---|
+| 1 | 1 | This configuration MUST NOT be used after 9/15/2025. |
+| 3 | 1 | This configuration MUST NOT be used after 9/15/2026. |
+| 4 | 2 | N/A |
+| N-2 | 2 | When the number of Network Perspectives used (i.e., “N”) is greater than or equal to (>=) six (6), a maximum of two (2) Network Perspectives MAY disagree with the Primary Domain Validation Determination. |
+
+Table: Defining Corroborating Domain Validation Evidence
```

Comment:
It would make me really happy if this table could disappear, and each definition of "corroborating evidence" be moved into the descriptions of each DCV method themselves. Maybe something like
```
##### 3.2.2.4.19 Agreed-Upon Change to Website - ACME
Confirming the Applicant's control over a FQDN by validating domain control of the FQDN using the ACME HTTP

<snip>

Validations using this method MUST implement Multi-Perspective Domain Validation as specified in Section 3.2.2.9. To count as corroborating, a Network Perspective MUST observe the same token (as defined in RFC 8555) as the Primary Domain Validation Determination.
```

Reply:
Attempted to move forward with the recommended approach in https://github.com/ryancdickson/staging/commit/cafdfc61365b962f7f88608009456af58ec7e123. Your feedback is appreciated.

While "spreadsheets are my love language" - I agree, there were too many tables.

Reply:
@aarongable - nudge. Would you consider the updated approach an improvement over the original (table-based format)?